kore/src/Kore/ModelChecker/Step.hs

```diff
@@ -183,13 +180,7 @@ transitionRule
                 then return (wrapper config)
                 else do
                     (Monad.Trans.lift . State.put) (Just ())
-                    trace
-                        (show . Pretty.vsep
-                            $ [ "config failed to prove the invariant:"
-                              , Pretty.indent 4 (unparse config)
-                              ]
-                        )
-                        return Unprovable
+                    return (Unprovable config)
```

Comment:
Could you add a test for returning unprovable configurations?

Reply:
Currently I don't have unit tests for KBMC. But I'll turn all the examples under https://github.com/kframework/kore/tree/master/src/main/k/in-progress/bmc/example1/check into integration tests. This can be done after this PR and https://github.com/kframework/k/pull/545 are merged.

Reply:
I'll approve this as-is, but I'll open a discussion about unit-tests on our main channel.